src/silx/gui/dialog/ColormapDialog.py

```diff
@@ -749,12 +755,26 @@ def _plotGammaMarkerConstraint(self, x, y):
             x = min(x, vmax)
         return x, y
 
+    def setDataInPlotMode(self, mode: str | DataInPlotMode):
```

Comment:
Maybe there is a better naming for: `setDataInPlotMode` and `DataInPlotMode`?
`PlotMode`? `DisplayMode`? `RangeDisplayMode`?

Reply:
+1 for `DisplayMode`

Reply:
But then I would also rename `DataInPlotMode` by `DisplayMode`

Reply:
@vallsv do you mind if we change `_DataInPlotMode` to `DisplayMode` ? As those are Enum we won't be able to deprecate them

Reply:
`_DataInPlotMode` is private, so no need for deprecation for me